client/src/mocks/components/auth/getLogin.js

```diff
@@ -0,0 +1,37 @@
+import * as original from '../../../../src/components/auth/getLogin.js';
+import sinon from 'sinon';
+
+export let init = function() {
+    return true;
+    // return original.login();
+}
+
+export let logout = original.logout;
+export let userInfo = original.userInfo;
+
+export let getAppAddresses = original.getAppAddresses;
+
+export function __stub(fnName, fn) {
+    console.log("stubbing", fnName);
+    switch(fnName) {
+        case 'init':
+        case 'login':
+            init = fn;
+            break;
+        case 'logout':
+            logout = fn;
+            break;
+        case 'userInfo':
+            userInfo = fn;
+            break;
+        case 'getAppAddresses':
+            getAppAddresses = fn;
+            break;
+        default:
+            init = fn;
+            logout = fn;
+            userInfo = fn;
+            getAppAddresses = fn;
+            break;
+    }
```

Comment:
Is there a reason we don't reference /auth/getLogin.js directly?

Reply:
This was done because I was having difficulty stubbing the functions. With this, since I "export let var1", it's quite easy to use the custom __stub function. I found sinon wasn't working.

I also did not use jest and jestdom to test because I felt the tests would be more robust if they were run in the browser, than in a simulated browser environment, where I would need to mock browser functions.


> 

- I have fixed the upload page.
- About the test bundle data I used the ones in the src/data folder. I haven't added new ones. 
I can do a meet this weekend.